mysql-test/suite/rocksdb/t/i_s_global_info.test

```diff
@@ -0,0 +1,15 @@
+--source include/have_rocksdb.inc
+
+SHOW CREATE TABLE INFORMATION_SCHEMA.ROCKSDB_GLOBAL_INFO;
+
+# Test that MYR-221 is addressed and that proper error is returned instead of a
+# crash when the SE is unloaded but the I_S plugin is still present
+
+UNINSTALL PLUGIN ROCKSDB;
+
+--error ER_PLUGIN_IS_NOT_LOADED
+SELECT * FROM INFORMATION_SCHEMA.ROCKSDB_GLOBAL_INFO;
```

Comment:
I'd try querying the rest of the I_S tables too here (no need for matching SHOW CREATE TABLE above, in fact I'd consider removing the existing one too)

Reply:
Well, this one was one guaranteed to crash where not all would, and it did not yet have some basic test for the table so I thought I would combine them. Really, this test should probably be executed on each i_s table for all existing tests, creating new tests for those missing i_s are well outside the scope.

Reply:
It's just that the added code for all the I_S tables needs to be checked, for coverage, copy-paste errors and the like.

But I have noticed the name of the testcase only now, so my suggestion would need a second testcase at least, not sure if it's worth the trouble

Reply:
Yeah, agreed, and maybe this is the opportunity to at least introduce the minimal tests for each i_s table that does just this check if no test exists and append this to any existing tests.